message-store/postgres/test/message-store-postgres.test.js

```diff
@@ -49,6 +49,36 @@ describe('message-store-postgres', () => {
     })
   })
 
+  describe('getStream', () => {
```

Comment:
For consistency with the other test cases, can you please use `get-stream` in this description.